.eslintrc.js

```diff
@@ -126,7 +139,8 @@ module.exports = {
       'test/helper.js',
     ],
     rules: {
-      'no-process-exit': 'off',
+      'node/no-process-exit': 'off',
+      'node/shebang': 'off',
```

Comment:
`node/shebang` requires that the shebang is only used for scripts identified in the `bin` field of `package.json`. I think our development scripts should be exempt from this rule.